### ISSUE TITLE
hub stack init with wrong stack id initialise new stack

### ISSUE THREAD

**Issue by TheRealSpaceShip** [agilestacks/hub-extensions#12]
Hub stack init extensions will init new stack if it was run with stack id which for example does not exists.

```
hub stack init some.fake.stack
```

On my opinion it should exit with non zero code. Same as if I will provider wrong state location with flag `-s`. 
Probably we should remove `-s` and use first argument as stack id or stack state location.

**Comment by akranga** [comment]
Resolved as part of commit: 2024b2086ffc295f51b3c1faa895f9ed2407729c

**Comment by TheRealSpaceShip** [comment]
In case of running `hub stack init some.stack.id` in Cloud Shell it produce and error

```shell
Initializing a new stack
* Using working directory: /home/igor_lysak/cloudshell_open/stacks/gke-empty-cluster
* Checking hub.yaml
parse error: Invalid numeric literal at line 1, column 4
```

This is because of execution of `hub state show some.stack.id -o json` in `hub-state-init` extension leads to an error

```shell
GCP Project ID is not set. Please do one of the following:
* re-run the command with --project flag
* set GOOGLE_PROJECT env variable
* set the Project ID using `gcloud config set project <project-id>` command
```

**Comment by TheRealSpaceShip** [comment]
Issue agilestacks/hub-state#1 is resolved. The issue with failed init of a stack before configure GCP resolved with 4b5585a
